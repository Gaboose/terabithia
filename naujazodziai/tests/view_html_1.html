<div class="view-html-nd">
  <h2>Bendroji informacija</h2>
  <ul class="description_list description_list--25-75">
    <li class="description_list__items">
      <div class="description_list__dt">Kirčiuotos formos:</div>
      <div class="description_list__dd">
        <span><span>naprotechnològas</span></span
        >, <span><span>naprotechnològė</span></span>
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Kirčiuotė: </div>
      <div class="description_list__dd description_accentuation">2</div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Rūšis: </div>
      <div class="description_list__dd description_kind">
        naujai skolintos šaknies žodis
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Žodžio klasė: </div>
      <div class="description_list__dd description_wordClass">skolinys</div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Kalbos dalis: </div>
      <div class="description_list__dd description_partOfSpeech">
        daiktavardis
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Kilmė: </div>
      <div class="description_list__dd">
        anglų,
        <span
          ><span><i>naprotechnologist</i>. <br /></span
        ></span>
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Giminiški naujažodžiai: </div>
      <div class="description_list__dd description_relatedRecord">
        <u
          ><a
            href=""
            data-ng-click="openVocabularyRecordByUuid({uuid:'7c8c32c9-8729-4b44-a683-acc7bd6466d9', homonymNo: '', senseNo: ''  })"
            >naprotechnologija<sup /></a
        ></u>
      </div>
    </li>
    <li class="description_list__items">
      <span class="description_list__dt">Pateikta: </span
      ><span class="description_list__dd">2025-03-27</span>
    </li>
  </ul>
  <h2>Reikšmė ir vartosena</h2>
  <ul class="description_list description_list--25-75">
    <li class="description_list__items">
      <div class="description_list__dt">Apibrėžtis: </div>
      <div class="description_list__dd description_definition">
        <span
          ><span>naprotechnologijų specialistas.<br /></span
        ></span>
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Vartojimo sritys: </div>
      <div class="description_list__dd description_usageField">
        medicina (sveikata), religija, technologijos
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Sąsajos: </div>
      <div class="description_list__dd description_linkedWord">
        <u
          ><a
            href=""
            data-ng-click="openVocabularyRecordByUuid({uuid:'7c8c32c9-8729-4b44-a683-acc7bd6466d9', homonymNo: '', senseNo: '1'  })"
            >naprotechnologija<sup /></a
        ></u>
      </div>
    </li>
  </ul>
  <h3>Vartosenos pavyzdžiai</h3>
  <h4>1-as pavyzdys</h4>
  <ul
    class="description_list description_list--small description_list--color-1 description_list--25-75"
  >
    <li class="description_list__items">
      <div class="description_list__dt">Pavyzdys: </div>
      <div class="description_list__dd description_example">
        <div>
          <span
            ><span
              ><span
                >Jau žinodamas tai, kas yra nevaisingumas, kaip dažnai jis
                pasitaiko ir kokios yra jo esminės priežastys, gydytojas pradeda
                ieškoti visų įmanomų būdų nevaisingumui pašalinti. Šiandienos
                reprodukcinė medicina gali pasiūlyti tris pagrindines pagalbos
                formas. Pirmosios iš jų esmė yra ta, kad normaliai sugyvenančios
                poros tinkamiausiam gyvybės pradėjimo metui nustatyti nuodugniai
                yra prižiūrimas moters mėnesinių ciklas. [...]</span
              ></span
            ></span
          >
        </div>
        <div>
          <span
            ><span
              ><span
                >Ir tai kone natūraliausias, be didesnio medicinos įsiterpimo,
                problemos sprendimo būdas. Šį, kaip nepažeidžiantį  jokių
                moralinių ar etinių vertybių, būdą aprobuoja ir patys gydytojai
                <b>naprotechnologai</b>.</span
              ></span
            ></span
          >
        </div>
        <div>
          <span
            ><span><span></span></span
          ></span>
        </div>
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Šaltinis: </div>
      <div class="description_list__dd description_source">
        <span>bernardinai.lt; aut. Tadeusz Wasilewski.<span></span></span>
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Šaltinio rūšis: </div>
      <div class="description_list__dd description_sourceType">
        internetinė žiniasklaida
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Metai: </div>
      <div class="description_list__dd description_year">2010</div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Kita metrika: </div>
      <div class="description_list__dd description_metrics">11 29.</div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Pavyzdį pateikė: </div>
      <div class="description_list__dd description_exampleSubmittedBy">
        Agnė Aleksaitė (2016 12 12).
      </div>
    </li>
  </ul>
  <h4>2-as pavyzdys</h4>
  <ul
    class="description_list description_list--small description_list--color-1 description_list--25-75"
  >
    <li class="description_list__items">
      <div class="description_list__dt">Pavyzdys: </div>
      <div class="description_list__dd description_example">
        <span
          ><span
            >Žymus akušeris ginekologas apsilankė
            <b>naprotechnologų</b> seminare: gydymo metodai šokiravo
            [antraštė]<br /></span
        ></span>
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Šaltinis: </div>
      <div class="description_list__dd description_source">
        <span
          >delfi.lt; aut. akušeris-ginekologas Rolandas Žiobakas.<span></span
        ></span>
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Šaltinio rūšis: </div>
      <div class="description_list__dd description_sourceType">
        internetinė žiniasklaida
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Metai: </div>
      <div class="description_list__dd description_year">2016</div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Kita metrika: </div>
      <div class="description_list__dd description_metrics">11 26.</div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Pavyzdį pateikė: </div>
      <div class="description_list__dd description_exampleSubmittedBy">
        Rita M.
      </div>
    </li>
  </ul>
  <h4>3-ias pavyzdys</h4>
  <ul
    class="description_list description_list--small description_list--color-1 description_list--25-75"
  >
    <li class="description_list__items">
      <div class="description_list__dt">Pavyzdys: </div>
      <div class="description_list__dd description_example">
        <span
          ><span
            >Žinote, Lietuva nėra visai išskirtinė savo reakcijomis į
            alternatyvius gydymo būdus šalis. Net ir šalyse, kur yra daugybė
            centrų ir klinikų, taikančių naprotechnologijos principus, kur
            gydymo rezultatai puikūs, pasitaiko įvairių reakcijų: ne tik
            diskreditavimas, ignoravimas, bet ir pačių medikų įsitraukimas į šią
            sritį. Tarkime, žinau vieną gydytoją ginekologą, kuris, anksčiau
            dirbęs pagalbinio apvaisinimo srityje, dabar yra puikus
            <b>naprotechnologas</b>.<br /></span
        ></span>
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Šaltinis: </div>
      <div class="description_list__dd description_source">
        <span
          >bernardinai.lt; aut. gydytoja akušerė-ginekologė Akvilė
          Esmantienė.<span></span
        ></span>
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Šaltinio rūšis: </div>
      <div class="description_list__dd description_sourceType">
        internetinė žiniasklaida
      </div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Metai: </div>
      <div class="description_list__dd description_year">2016</div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Kita metrika: </div>
      <div class="description_list__dd description_metrics">12 05.</div>
    </li>
    <li class="description_list__items">
      <div class="description_list__dt">Pavyzdį pateikė: </div>
      <div class="description_list__dd description_exampleSubmittedBy">
        Agnė Aleksaitė (2016 12 12).
      </div>
    </li>
  </ul>
</div>
