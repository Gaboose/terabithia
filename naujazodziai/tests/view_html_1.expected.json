{
  "Bendroji informacija": {
    "Kirčiuotos formos": "naprotechnològas, naprotechnològė",
    "Kirčiuotė": "2",
    "Rūšis": "naujai skolintos šaknies žodis",
    "Žodžio klasė": "skolinys",
    "Kalbos dalis": "daiktavardis",
    "Kilmė": "anglų, naprotechnologist.",
    "Giminiški naujažodžiai": "naprotechnologija",
    "Pateikta": "2025-03-27"
  },
  "Reikšmė ir vartosena": {
    "Apibrėžtis": "naprotechnologijų specialistas.",
    "Vartojimo sritys": "medicina (sveikata), religija, technologijos",
    "Sąsajos": "naprotechnologija",
    "Vartosenos pavyzdžiai": {
      "1-as pavyzdys": {
        "Pavyzdys": "Jau žinodamas tai, kas yra nevaisingumas, kaip dažnai jis pasitaiko ir kokios yra jo esminės priežastys, gydytojas pradeda ieškoti visų įmanomų būdų nevaisingumui pašalinti. Šiandienos reprodukcinė medicina gali pasiūlyti tris pagrindines pagalbos formas. Pirmosios iš jų esmė yra ta, kad normaliai sugyvenančios poros tinkamiausiam gyvybės pradėjimo metui nustatyti nuodugniai yra prižiūrimas moters mėnesinių ciklas. [...] Ir tai kone natūraliausias, be didesnio medicinos įsiterpimo, problemos sprendimo būdas. Šį, kaip nepažeidžiantį jokių moralinių ar etinių vertybių, būdą aprobuoja ir patys gydytojai naprotechnologai.",
        "Šaltinis": "bernardinai.lt; aut. Tadeusz Wasilewski.",
        "Šaltinio rūšis": "internetinė žiniasklaida",
        "Metai": "2010",
        "Kita metrika": "11 29.",
        "Pavyzdį pateikė": "Agnė Aleksaitė (2016 12 12)."
      },
      "2-as pavyzdys": {
        "Pavyzdys": "Žymus akušeris ginekologas apsilankė naprotechnologų seminare: gydymo metodai šokiravo [antraštė]",
        "Šaltinis": "delfi.lt; aut. akušeris-ginekologas Rolandas Žiobakas.",
        "Šaltinio rūšis": "internetinė žiniasklaida",
        "Metai": "2016",
        "Kita metrika": "11 26.",
        "Pavyzdį pateikė": "Rita M."
      },
      "3-ias pavyzdys": {
        "Pavyzdys": "Žinote, Lietuva nėra visai išskirtinė savo reakcijomis į alternatyvius gydymo būdus šalis. Net ir šalyse, kur yra daugybė centrų ir klinikų, taikančių naprotechnologijos principus, kur gydymo rezultatai puikūs, pasitaiko įvairių reakcijų: ne tik diskreditavimas, ignoravimas, bet ir pačių medikų įsitraukimas į šią sritį. Tarkime, žinau vieną gydytoją ginekologą, kuris, anksčiau dirbęs pagalbinio apvaisinimo srityje, dabar yra puikus naprotechnologas.",
        "Šaltinis": "bernardinai.lt; aut. gydytoja akušerė-ginekologė Akvilė Esmantienė.",
        "Šaltinio rūšis": "internetinė žiniasklaida",
        "Metai": "2016",
        "Kita metrika": "12 05.",
        "Pavyzdį pateikė": "Agnė Aleksaitė (2016 12 12)."
      }
    }
  }
}